pootle/settings/60-translation.conf

```diff
@@ -52,35 +52,9 @@ POOTLE_MT_BACKENDS = [
 # the trailing slash.
 AMAGAMA_URL = 'https://amagama-live.translatehouse.org/api/v1/'
 
-# Local Translation Memory service
-# Setup for connection to the ElasticSearch server for translation memory based
-# on the projects hosted on Pootle.
-# You may want to set AMAGAMA_URL to '' (empty string) if using this service.
-POOTLE_TM_SERVER = {
-    'local': {
-        'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
-        'HOST': 'localhost',
-        'PORT': 9200,
-        # Every TM server must have its own unique index.
-        'INDEX_NAME': 'translations',
-        # Provides a weighting factor to alter the final score for TM results
-        # from this TM server. Valid values are between ``0.0`` and ``1.0``,
-        # both included. Defaults to ``1.0`` if not provided.
-        'WEIGHT': 1.0,
-        # For valid values for MIN_SCORE see
-        # http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/common-options.html#fuzziness
-        'MIN_SCORE': 'AUTO',
-    },
-    'external': {
-        'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
-        'HOST': 'localhost',
-        'PORT': 9200,
-        'INDEX_NAME': 'external-translations',
-        'WEIGHT': 0.9,
-        'MIN_SCORE': 'AUTO',
-    },
-}
-
+# See 90-local.conf.template or default pootle.ini for example of local TM
```

Comment:
good point - i think ill remove the ref to that file altogether - and just keep the ref to 90-local